lib/glific/partners.ex

```diff
@@ -932,9 +932,14 @@ defmodule Glific.Partners do
   end
 
   defp credential_update_callback(organization, credential, "maytapi") do
-    WAManagedPhones.fetch_wa_managed_phones(organization.id)
-    WAGroups.fetch_wa_groups(organization.id)
-    {:ok, credential}
+    case WAManagedPhones.fetch_wa_managed_phones(organization.id) do
+      {:error, error} ->
```

Comment:
We could use `with` clause to avoid the `{:error, error}` duplicate right?. What you think?

Reply:
fixed this. Initially I used with statement only but it was giving dialyzer error. Strangely it is not throwing error now